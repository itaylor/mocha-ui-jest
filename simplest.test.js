describe('simplest test', () => {
  test('expect works', () => {
    expect(true).toBe(true);
    expect([{ that: 'this' }]).toMatchSnapshot();
  });

  test('multiple snapshots in one test work', () => {
    expect({ foo: 'bar' }).toMatchSnapshot();
    expect({ bar: 'foo' }).toMatchSnapshot();
  });
});
