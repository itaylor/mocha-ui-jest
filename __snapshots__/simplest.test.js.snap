// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simplest test expect works 1`] = `
Array [
  Object {
    "that": "this",
  },
]
`;

exports[`simplest test multiple snapshots in one test work 1`] = `
Object {
  "foo": "bar",
}
`;

exports[`simplest test multiple snapshots in one test work 2`] = `
Object {
  "bar": "foo",
}
`;
